apps/api-gateway/src/chat/chat.module.ts

```diff
@@ -0,0 +1,26 @@
+import { Module } from '@nestjs/common';
+import { ClientProxyFactory, Transport } from '@nestjs/microservices';
+import { ChatController } from './chat.controller';
+import { ChatService } from './chat.service';
+
+@Module({
+  exports: [ChatService],
+  controllers: [ChatController],
+  providers: [
+    ChatService,
+
+    {
+      provide: 'CHAT_SERVICE',
+      useFactory: () => {
+        return ClientProxyFactory.create({
+          transport: Transport.TCP,
+          options: {
+            host: 'localhost',
+            port: Number(process.env.CHAR_PORT),
```

Comment:
```suggestion
            port: Number(process.env.CHAT_PORT),
```
해당 ENV 값의 오타가 있는 것 같습니다!

Reply:
엇 오타가 있었군요. 바로 수정하겠습니다!